Core/FunctionTree/ParameterList.hpp

```diff
@@ -78,91 +82,86 @@ class ParameterList {
 
   // Value
   // Single sized values
-  virtual std::shared_ptr<ComPWA::Value<int>> intValue(size_t i) {
+  virtual std::shared_ptr<Value<int>> intValue(size_t i) {
```

Comment:
In the header files we should stick to the ful namespace.

Reply:
Whats the reasoning behind this?